addons/payment_paypal/__manifest__.py

```diff
@@ -16,5 +16,6 @@
     ],
     'application': True,
     'uninstall_hook': 'uninstall_hook',
+    'auto_install': True,
```

Comment:
Do we want yet another auto-install module?

Reply:
As this module and payment_stripe will always be installed during the sale/website_sale onboarding (step payment acquirer), it has been discussed to make those modules `auto_install=True`.

It makes the onboarding more smooth and faster but, yet, it may be marginal compared to the waste of time for every other instances which don't go through the onboarding.

Reply:
Both points of view make sense. It makes the onboarding of Sales, eCommerce, etc. faster BUT these modules will install in pretty much every DB since they depend on `payment` with is an auto-install dependency of `account` which is installed by almost all apps...

It's maybe safer to remove the `auto_install` for now and see how badly it impacts the onboarding.